portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,8 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Jess's Portfolio</h1>
+      <p>If you're here, you probably already know that I'm a <strong>Google STEP intern</strong>. However, I'm also frequently described as an <strong>Incoming Sophomore at UT Austin</strong>, a <strong>CS and Math Double Major</strong>, a <strong>fraternal twin</strong>, an <strong>amateur unicyclist</strong>, and a <strong>tennis enthusiast</strong>. When I work, I describe myself as a <strong>curious developer</strong>. In the future, I hope to become a <strong>decent ukulelist</strong>, a <strong>strong writer</strong>, and eventually, a <strong>full-time software developer</strong>.
```

Comment:
Can you play *Somewhere Over the Rainbow*?

Reply:
I used to be able to, but it's been a while since I've played. This summer I plan on getting back into it!